test_nb.ipynb

```diff
@@ -2,12 +2,13 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        .agg({'score'     :['mean',list]})
</pre><p>Why did you change this to mean?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/justinek/review_notebooks/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Oh because it's better than median.</p>